frontend/components/CalendarFull.vue

```diff
@@ -0,0 +1,715 @@
+                            },
+                        );
+                    }
+                });
+                alreadyAdded.value = true;
+            }
+        },
+        { immediate: true },
+    );
+});
+
+async function initializeDrop(info: EventObject) {
+    const calApi = fullCalendar.value.getApi();
+    if (info.event._def.extendedProps.defId === undefined) {
+        editDrop(info);
+        return;
+    }
+    const activityId = info.event._def.extendedProps.defId;
+    const startTime = info.event._instance.range.start.toISOString();
+    const endTime = info.event._instance.range.end.toISOString();
+    const activity = {
+        start: startTime.substring(0, startTime.length - 2),
+        end: endTime.substring(0, endTime.length - 2),
+    };
+
+    await client(
+        `/api/journey/${props.id}/activity/${activityId}/calendarActivity/`,
+        {
+            method: "POST",
+            body: activity,
+            async onResponse({ response }) {
+                if (response.ok) {
+                    toast.add({
+                        severity: "success",
+                        summary: t.value("calendar.add.toast.success.heading"),
+                        detail: t.value("calendar.add.toast.success.detail"),
+                        life: 6000,
+                    });
+                    const activities = store.activityData as Activity[];
+                    activities.forEach((activity: Activity) => {
+                        if (activity.id == response._data.activity_id) {
+                            response._data.title = activity.name;
+                            calApi.addEvent(response._data);
+                        }
+                    });
+                }
+            },
+            async onRequestError() {
+                toast.add({
+                    severity: "error",
+                    summary: t.value("common.toast.error.heading"),
+                    detail: t.value("common.error.unknown"),
+                    life: 6000,
+                });
+            },
+            async onResponseError() {
+                toast.add({
+                    severity: "error",
+                    summary: t.value("common.toast.error.heading"),
+                    detail: t.value("common.error.unknown"),
+                    life: 6000,
+                });
+            },
+        },
+    );
+    info.event.remove();
+}
+
+async function editDrop(info: EventObject) {
+    const activityId = info.event._def.extendedProps.activity_id;
+    const id = ref(info.event._def.publicId);
+    const startTime = info.event._instance.range.start.toISOString();
+    const endTime = info.event._instance.range.end.toISOString();
+
+    const activity = {
+        start: startTime.substring(0, startTime.length - 2),
+        end: endTime.substring(0, endTime.length - 2),
+    };
+
+    await client(
+        `/api/journey/${props.id}/activity/${activityId}/calendarActivity/${id.value}`,
+        {
```

Comment:
Die Aktivitäten verschwinden auch nicht aus dem Aktivitäten Pool wenn man diese in den Kalender zieht oder manuell einplant